controllers/admin/AdminSearchController.php

```diff
@@ -264,43 +264,21 @@ public function searchFeatures()
     {
         $this->_list['features'] = [];
 
-        global $_LANGADM;
```

Comment:
You removed the use of $_LANGDAM. Can you confirm that $_LANGDAM can safely be removed? Can you explain what it is used for, and why we can remove it? In your PR, your search implementation seems a lot simpler than current implementation